src/travel-details-screens/components/FlexibleTransportBookingDetails.tsx

```diff
@@ -0,0 +1,201 @@
+      </TouchableOpacity>
+
+      <ScrollView
+        style={[
+          style.contentContainer,
+          {
+            marginBottom: Math.max(safeAreaBottom, theme.spacings.medium),
+          },
+        ]}
+      >
+        <View style={style.messageBoxContainer}>
+          <FlexibleTransportMessageBox
+            leg={leg}
+            publicCode={publicCode}
+            now={now}
+            showStatusIcon={true}
+          />
+        </View>
+
+        {!isSmallScreen && (
+          <View style={style.imageContainer}>
+            <FlexibleTransport height={107} />
+          </View>
+        )}
+
+        <ThemeText type="heading__title" style={style.title}>
+          {t(TripDetailsTexts.flexibleTransport.contentTitle(publicCode))}
+        </ThemeText>
+        <View style={style.steps} accessibilityRole="list">
+          {
+            // eslint-disable-next-line rulesdir/translations-warning
+            TripDetailsTexts.flexibleTransport.steps.map((step, i) => {
+              const stepNumberText = `${i + 1}. `;
+              const stepInstructionText = t(step);
+              return (
+                <View
+                  key={i}
+                  style={style.step}
+                  accessible={true}
+                  accessibilityLabel={stepNumberText + stepInstructionText}
+                  accessibilityRole="text"
+                >
+                  <ThemeText type="body__primary">{stepNumberText}</ThemeText>
+                  <ThemeText type="body__primary">
+                    {stepInstructionText}
+                  </ThemeText>
+                </View>
+              );
+            })
+          }
+        </View>
+
+        <TouchableOpacity
+          style={style.readMoreAbout}
+          onPress={() => {
+            const flexTransportInfoUrl =
+              getTextForLanguage(
+                configurableLinks?.flexTransportInfo,
+                language,
+              ) || '';
+            flexTransportInfoUrl && Linking.openURL(flexTransportInfoUrl);
+          }}
+          accessibilityRole="link"
+        >
+          <ThemeText
+            color="secondary"
+            style={style.linkText}
+            type="body__primary--underline"
+          >
+            {t(TripDetailsTexts.flexibleTransport.readMoreAbout(publicCode))}
+          </ThemeText>
+        </TouchableOpacity>
+
+        {bookingIsAvailable && <FlexibleTransportBookingOptions leg={leg} />}
+
+        {Platform.OS === 'android' && (
+          <View // scroll bugfix
+            style={{height: theme.spacings.xLarge}}
+          />
+        )}
+      </ScrollView>
+    </BottomSheetContainer>
+  );
+};
+
+const useStyle = StyleSheet.createThemeHook((theme) => ({
+  title: {
+    paddingHorizontal: theme.spacings.small,
+    paddingTop: theme.spacings.small,
+  },
+  closeIconContainer: {
+    position: 'absolute',
+    top: theme.spacings.large / 2 + theme.spacings.small,
```

Comment:
What does this solve? why not then another value like `theme.spacing.small` or another value from the spacing values in the theme? Instead of `theme.spacings.large / 2`.